src/react/App.css

```diff
@@ -109,13 +109,13 @@ body {
   justify-content: center;
 }
 
-.text {
+.popup_header-text {
```

Comment:
not to be too picky, but this should have two underscores "popup__header". This system is called [BEM](https://css-tricks.com/bem-101/) :)

Reply:
Good convention 
added it somewhere else too :)

Reply:
I'll adress the others with the color topic

Reply:
I'll also remove the rest of the index.css with the color pull request